File: Exercícios/Capitulo_2/Resolvidos/questão_2_cap_2.py
deslocamento = float(input("Informe a variação de deslocamento do objeto em metros (m): "))
var_tempo = float(input("Informe a variação de tempo do movimento em segundos (s): "))

vel_med = deslocamento/var_tempo

print(f"Velocidade média = {vel_med:.2f} m/s")

File: Exercícios/Capitulo_2/Resolvidos/questão_2cap2.cpp
#include <iostream>
#include <locale.h>
using namespace std;

int main()
{
    setlocale(LC_ALL, "Portuguese");

    float deslocamento, var_tempo, vel_med;

    cout << "Informe a variação do deslocamento: ";
    cin >> deslocamento;

    cout << "Informe a variação de tempo do movimento: ";
    cin >> var_tempo;

    vel_med = deslocamento / var_tempo;

    cout << "\nVelocidade media = " << vel_med << " m/s";

    return 0;
}